notebooks/analyze_model_output.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>Is firestore connection still necessary here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>nope not needed</p><p> </p>